### PR TITLE
The 1st large cleanup of Social filters

### PR DESCRIPTION
**Pull request by Yuki2718**
```
! ---------- Redundant
hurriyet.com.tr###socialMediaShare
aliexpress.ru##div[data-role="sns-share"]

ambarexpress.com##.footer-whatsapp-fix

! in 2 files
indirimdemarket.com#$##social-media-modal { display: none!important; }
indirimdemarket.com#$#body { overflow: visible!important; }

! in 2 files
/pandalocker.$script,domain=lanka.ru

windowstechblog.nl#$#.onp-sl-blur-area[style="filter: blur(5px);"] { filter: none!important; }
warezcrack.net#$#.onp-sl-content > [data-lock-id^="onpLock"] { display: block!important; }

! ~facebook.com##a[href^="fb-messenger://share"]
~facebook.com##a[href^="fb-messenger://share/"]

! ~hatena.ne.jp##a[href^="http://b.hatena.ne.jp/add?"]
~hatena.ne.jp##a[href^="http://b.hatena.ne.jp/add?mode=confirm&url="]

! ~pinterest.com##a[href^="https://pinterest.com/pin/create/"]
~pinterest.com##a[href^="https://pinterest.com/pin/create/link/?"]
~pinterest.com##a[href^="https://pinterest.com/pin/create/button/"]

! screenshot.ru##.panel-share-networks
screenshot.ru##.panel-share-networks > ul

! townhall.com##.btn-twitter-lg
townhall.com##button.btn-twitter-lg

! rtl.de,vox.de##.rtli-master-social-share
rtlnext.rtl.de##body .rtli-master-social-share

! lostfilm.info##.news-container > div.left+div.right
lostfilm.info##.news-container > div[class="left"][style="width: 400px;"]+div.right

! foxbaltimore.com,foxchattanooga.com,foxillinois.com,foxlexington.com,foxnebraska.com,foxreno.com,foxrochester.com,foxsanantonio.com,nbcmontana.com,okcfox.com,siouxlandnews.com,wach.com,wchstv.com,wcyb.com,wfxl.com,wgxa.tv,wlos.com,wsbt.com,wsmh.com,wvah.com,cbs2iowa.com,fox4beaumont.com,mycbs4.com,mynews4.com,news4sanantonio.com,abc6onyourside.com,wtov9.com,turnto10.com,fox11online.com,mynbc15.com,nbc16.com,abc22now.com,fox23maine.com,nbc24.com,fox28media.com,myfox28columbus.com,fox42kptm.com,fox45now.com,fox56.com##div[id^="js-Social-Share"]
nbcmontana.com,mynews4.com,nbc16.com,turnto10.com,news4sanantonio.com,nbc24.com,mycbs4.com,wtov9.com,mynbc15.com##div[id^="js-Social-Share-"]

! haberturk.com##.share
m.haberturk.com##.share > li > .icon-bip
m.haberturk.com##.share > li > .icon-email

! oggito.com##body .facebookBtn
oggito.com##body div.facebookBtn

! pg11.ru,pg12.ru,pg13.ru,pg21.ru,prochepetsk.ru,progorod33.ru,progorod43.ru,progorod58.ru,progorod59.ru,progorod62.ru,progorod76.ru,progorodnn.ru,progorodnsk.ru,progorodsamara.ru,progoroduhta.ru##.article__likes
pg11.ru##.article__likes
progorodnn.ru,progorod62.ru##.article__likes

! mobil.mz-web.de##.recommend-article
mobil.mz-web.de##.recommend-article.sharing

! nasa.gov##.chertools_floater
climate.nasa.gov##.chertools_floater[data-react-props*="share"]

! medindia.net,blog.malwarebytes.com##.share-section
blog.malwarebytes.com###content > .share-section

! mobile01.com##a[onclick^="share_"]
mobile01.com##a[href="javascript:void(0);"][onclick^="share_"]

! thehindubusinessline.com##.aricle-share > li[data-category="social-shares"]
thehindubusinessline.com##.share-topic > ul.aricle-share > li[data-category="social-shares"]

! videogameschronicle.com##.vgc-socialshare
videogameschronicle.com##div.vgc-socialshare

! dailymail.co.uk##.social > ul
dailymail.co.uk##.social > ul > li:not([class="comments-count"])

! estadao.com.br##.share
estadao.com.br##.module__share > ul.share:first-of-type

! care2.com##.share
care2.com##.share.side_float
care2.com##.share > a[href="#"]:not([class="email"])
care2.com##.right_side > div.share

podrobnosti.ua##.twitter-timeline

! techspot.com##.social-column
techspot.com##body .social-column.mobile-only-social

! businessinsider.com,postnauka.ru,spletnik.ru,dreamtv.com.tr,theatlantic.com,fr.de,cozumpark.com,acquavivalive.it##.share-wrapper
businessinsider.com##.share-wrapper > .share-more

! cracxpro.com,hackintoshzone.com,gigadownloads.co,techsini.com,lookdevices.ru##.onp-sl-overlap-box
gigadownloads.co,techsini.com##.onp-sl-overlap-box

! sony.ru,otus.ru,daily.afisha.ru,about.com,drive2.com,drive2.ru,metrolyrics.com,surfingbird.ru##.js-share
drive2.com,drive2.ru##.o-group.o-group--m.js-share


! ---------- Dead domain (all checked with real paths)
||widget.instagramm.ru^$third-party
||widgets.instela.com/widget.php?url=$third-party
||cim.meebo.com^$third-party
||i.po.st^$third-party
@@||i.po.st^$domain=rolandgarros.com
@@||i.po.st/static/v3/post-widget.js$domain=seenive.com
||img-trkcllcropy.mncdn.com^$third-party
||iknopo.ru^$third-party
||socpost.net^$third-party
||static.ak.connect.facebook.com/connect.php$third-party
||widgets.digg.com/$domain=~digg.com
||widgets.twimg.com/$third-party


||mblogi.qip.ru/knopka$third-party
futboltv24.net##.trd-text > a[href="paylas"]
||futboltv24.net/social.html
```

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -705,7 +700,6 @@ kp.ru##.sharing_wrapper
 pdf.movavi.ru##.article-buttons
 blacksilk.sclub.com.tw##div[onmouseover^="SHARE_"]
 blog.remarkable.com##button > svg[width="29"][height="29"]
-podrobnosti.ua##.twitter-timeline
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:708]: `podrobnosti.ua##.twitter-timeline` is required
image

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -127,8 +127,6 @@ rtvi.com##.soc
 m-cacm.acm.org##.social-links_wrapper
 cacm.acm.org##.fav_bar > a#sendByEmail
 cacm.acm.org##.fav_bar > a[original-title^="Share"]
-indirimdemarket.com#$##social-media-modal { display: none!important; }
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:130]: `! in 2 files`
if in Social + Annoyances=>popups, it is OK(for all removed rules)

**Review by Alex-302 — CHANGES_REQUESTED** [review]
In general - good. Thanks.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:708]: The same rule is already in the common section.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:130]: I mean the same rule is in a different file of Social filters.

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply SocialFilter/sections/specific.txt:708]: social locker rules better leave in that group

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:708]: The rule is not in the social locker group?

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:130]: Should I revert, or will you?

Comment (Alex-302) [reply SocialFilter/sections/specific.txt:130]: No, leave as is.

**Comment by Alex-302** [comment]
Thanks.

**Pull request merged**